python/graphframes/tests.py

```diff
@@ -376,93 +413,101 @@ def test_parallel_personalized_page_rank(self):
         maxIter = 15
         sourceIds = [1, 2, 3, 4]
         pr = g.parallelPersonalizedPageRank(resetProb, sourceIds=sourceIds, maxIter=maxIter)
-        self._hasCols(pr, vcols=['id', 'pageranks'], ecols=['src', 'dst', 'weight'])
+        self._hasCols(pr, vcols=["id", "pageranks"], ecols=["src", "dst", "weight"])
 
     def test_shortest_paths(self):
         edges = [(1, 2), (1, 5), (2, 3), (2, 5), (3, 4), (4, 5), (4, 6)]
+        # Create bidirectional edges.
         all_edges = [z for (a, b) in edges for z in [(a, b), (b, a)]]
-        edges = self.spark.createDataFrame(all_edges, ["src", "dst"])
+        edgesDF = self.spark.createDataFrame(all_edges, ["src", "dst"])
```

Comment:
this is not the same 
edges != edgesDF

Reply:
Sorry, you don't like the use of `edgesDF` variable name? The edges are defined right there a few lines above.

Reply:
@bjornjorgensen fixed